File: malevich-server/src/main/java/io/malevich/server/transfer/TransactionsDto.java
package io.malevich.server.transfer;

import lombok.Getter;
import lombok.Setter;


@Getter
@Setter
public class TransactionsDto {

    private Long id;

    private TransactionTypeDto type;

    private java.sql.Timestamp effectiveDate;

    private CounterpartyDto party;

    private CounterpartyDto counterparty;

    private ArtworkStockDto artworkStock;

    private Double amount;

    private Integer quantity;

}

File: malevich-web/src/app/_transfer/transactionsDto.ts
import {ArtworkStockDto} from "./artworkStockDto";
import {CounterpartyDto} from "./counterpartyDto";
import {TransactionTypeDto} from "./transactionTypeDto";

export class TransactionsDto {
  id: number;
  type: TransactionTypeDto;
  effectiveDate: Date;
  party: CounterpartyDto;
  counterparty: CounterpartyDto;
  artworkStock: ArtworkStockDto;
  amount: number;
  quantity: number;
}
